qiita_db/portal.py

```diff
@@ -338,12 +350,28 @@ def add_analyses(self, analyses):
         QiitaDBError
             Some given analyses do not exist in the system,
             or are default analyses
+            Portal does not contain all studies used in analyses
         QiitaDBWarning
             Some analyses already exist in the given portal
         """
         self._check_analyses(analyses)
 
         conn_handler = SQLConnectionHandler()
+        # Make sure new portal has access to all studies in analysis
```

Comment:
@biocore/qiita-dev comments?

@squirrelo an SQL query is not very cheap compared with an if statement....